jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -61,14 +61,15 @@
     "\n",
     "#### What does a change detection result represent?\n",
     "\n",
-    "In the building construction feed, a change result contains a geographic polygon that represents the footprint of a contiguous area where we have detected building construction. \n",
-    "Thus, these polygons represent general built-up area, and may not identify individual buildings. They may be a group of small buildings clumped together, or a new adjacent wing of an existing building.\n",
+    "In the building construction feed, a change result contains a geographic polygon or grid cell that represents the footprint of a contiguous area where we have detected building construction. Thus, these polygons represent general built-up area, and may not identify individual buildings. They may be a group of small buildings clumped together, or a new adjacent wing of an existing building.\n",
     "\n",
-    "In the case of the road construction feed, the change results contain a linestring, instead of a polygon. This linestring represents the skeleton of a road segment that has been constructed. \n",
+    "In the case of the road construction feed, the change results also contain a polygon or grid cell. It represents the footprint of a contiguous area where a road segment has been constructed.\n",
     "\n",
     "#### When did the change actually occur?\n",
     "\n",
-    "Building and road construction is typically detected 3-4 months after the event took place. Our system performs statistical inference on the time series of images at the pixel level, and issues a change detection once there is enough confidence that a true transition occurred. Since we use images from Planet Basemaps, which have a monthly sampling rate, we cannot determine the week or day of an event. \n"
+    "Building construction is typically detected an average of two months after the event took place, while road construction is detected an average of two and half months after the event took place. Our system performs statistical inference on the time series of images at the pixel level, and issues a change detection once there is enough confidence that a true transition occurred. Currently, we use images from Planet's Monthly Basemaps, which have a monthly sampling rate. This results in an average lag of two to two and half month lag and a monthly cadence. \n",
```

Comment:
Average lag is useful to know, but can we better describe the distribution of lags (or at least the 95th percentile worst case scenario or something like that)?

Reply:
I believe Jonathan wants to keep very specific metrics like that (% precision, % recall, distribution of lag) out of user facing docs for now, with the results more general and qualitative.